managing-os/templates/archlinux.md

```diff
@@ -204,20 +210,13 @@ Main maintainer of this template is [Olivier Médoc](mailto:o_medoc@yahoo.fr).
 <br>
 <br>
 
-        *   Screen '**Get Resources**' wants to download additional packages needed for the choosen plugin/s.
+        *   Screen '**Get sources**' wants to download additional packages needed for the choosen plugin/s.
         
-            *   Select '**OK**' Prss '**Enter**'
-            
-        *   Upon completion you will get choose '**OK**' to proceed to the next screen
-        
-            *   Press '**Enter**'
+            *   Select '**No**' Press '**Enter**'
```

Comment:
Sorry I should have been more verbose about it.

This(+ setting _GIT_URL_builder_archlinux_ later) is actually the main reason for the whole change.
Choosing "Yes" here tries to fetch "qubes-builder-archlinux.git" from main QubesOS repository. As it does not exist, it asks for username/password for github(probably to check whether private repo with such name exists). Firstly this is at least confusing, considering that user tries to download from public repo. And the whole downloading step fails afterwards as it cannot access repo.

Additionally as far as I understand this step is done anyway with _make get-sources_, which is done after manually fixing _builder.conf_ and works like a charm.